Kernel/Process.cpp

```diff
@@ -64,6 +64,7 @@ static Lock* s_hostname_lock;
 static VirtualAddress s_info_page_address_for_userspace;
 static VirtualAddress s_info_page_address_for_kernel;
 VirtualAddress g_return_to_ring3_from_signal_trampoline;
+Region* g_return_to_ring3_from_signal_trampoline_region;
```

Comment:
There's no need to keep this `Region` in a global variable, as it has no outside clients.

Reply:
Fixed.